features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -14,14 +14,18 @@ struct DisplacementParams
 
 namespace ExtendedMaterials
 {
-	float AdjustDisplacement(float displacement, DisplacementParams params)
+	float DisplacementHeight0(float displacement, DisplacementParams params)
 	{
-		return (displacement - 0.5) * params.DisplacementScale + 0.5 + params.DisplacementOffset;
+		return (displacement - 0.5) * params.HeightScale + params.DisplacementOffset;
```

Comment:
So now `HeightScale` is applied twice? First to `maxHeight` and then here

Reply:
The actual height is needed in GetTerrainHeight, it is divided back to normalized value in GetParallaxCoords.

Reply:
Hmmm maybe that's okay, need to think about it more. But `DisplacementOffset` certainly shouldn't be here. Well, actually for everything to be completely correct you'd to do `(Displacement01(displacement) - 0.5) * HeightScale` here, but since `DisplacementOffset` and `DisplacementScale` are trivial for landscape `Displacement01` can be safely skipped here